docs/example-scenario/apps/secure-deploy-to-app-svc-env.md

```diff
@@ -0,0 +1,175 @@
+3. User request then flows through the ILB of the ASE, which in turn routes the traffic to the Contoso Expenses Web App.
+4. User then proceeds to create an Expense Report
+5. As part of the expense creation, the deployed API App is invoked to retrieve user's manager name and email
+6. The Created Expense Report is stored in Azure SQL Database
+7. To facilitate continuous deployments, code gets checked into the Azure DevOps instance
+8. The build VM has the Azure DevOps Agent installed
+9. This allows the build VM to pull the bits for the Web App to deploy to the ASE (as the Build VM is deployed in a Subnet inside the same VNET)
+
+### Components
+
+* ILB [App Service Environment][intro-to-app-svc-env] The Azure App Service Environment is an Azure App Service feature that provides a fully isolated and dedicated environment for securely running App Service apps at high scale. 
+* [Isolated App Service Plan Pricing Tier][isolated-tier-pricing-and-ase-pricing] The Isolated service plan is designed to run mission critical workloads, that are required to run in a virtual network. The Isolated plan allows customers to run their apps in a private, dedicated environment in an Azure datacenter using Dv2-series VMs with faster processors, SSD storage, and double the memory-to-core ratio compared to Standard
+* Azure App Services [Web App][docs-webapps] and [API App][docs-apiapps] hosts web applications and RESTful APIs allowing autoscale and high availability without having to manage infrastructure.
+* [Application Gateway][docs-appgw] Azure Application Gateway is a web traffic load balancer that enables you to manage traffic to your web applications.
+* [Web Application Firewall][docs-waf] Web application firewall (WAF) is a feature of Application Gateway that provides centralized protection of your web applications from common exploits and vulnerabilities.
+* [Azure SQL Database][docs-sql-database] Azure SQL Database is a relational database-as-a-service (DBaaS) based on the latest stable version of Microsoft SQL Server Database Engine.
+* [Azure Networking] Azure provides a variety of networking capabilities that can be used together or separately
+* [Azure DevOps][docs-azure-devops] Azure DevOps Services provides development collaboration tools including high-performance pipelines,
+free private Git repositories, configurable Kanban boards, and extensive automated and cloud-based load testing
+* [Build Azure VM][docs-azure-vm] In Azure Portal, create a Visual Studio Enterprise 2017 (latest release) on Windows Server 2016 (x64) VM that will be used as the build agent. 
+
+### Alternatives
+- [Service Fabric][docs-service-fabric] - A platform focused around building distributed components that benefit from being deployed and run across a cluster with a high degree of control. Service Fabric can also be used to host containers.
```

Comment:
Containers being used anywhere in this?  Why was service fabric or Kuberneties not chosen?